src/core/agent/services/keriaNotificationService.ts

```diff
@@ -294,6 +340,55 @@ class KeriaNotificationService extends AgentService {
     }
   }
 
+  async retryFailedNotifications() {
+    const failedNotifications = await this.basicStorage.findById(
+      MiscRecordId.FAILED_NOTIFICATION
+    );
+
+    if (!failedNotifications || !failedNotifications.content) return;
+
+    const { notificationId, attempts, lastAttempt } =
+      failedNotifications.content as unknown as NotificationAttempts;
+    const now = Date.now();
+
+    const backoffDelays = [5000, 10000, 30000, 60000, 300000];
+    const delay =
+      backoffDelays[Math.min(attempts - 1, backoffDelays.length - 1)];
+
+    if (now - lastAttempt >= delay) {
```

Comment:
This all needs to change because the attempt is PER notification. Not for all notifications.